python/ray/_private/services.py

```diff
@@ -1211,8 +1208,7 @@ def start_api_server(
             if we fail to start the API server. Otherwise it will print
             a warning if we fail to start the API server.
         host: The host to bind the dashboard web server to.
-        gcs_address: The gcs address the dashboard should connect to
-        cluster_id_hex: Cluster ID in hex.
+        gcs_client: The client to GCS.
```

Comment:
In the docstring, let's note that the dashboard-startup timeout is now configurable.